### ISSUE TITLE
Formatting is strange on the create plan page 

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2500]
The formatting is not working well - the guidance is appearing over the text/questions and it is not possible to hide this. 

<img width="1182" alt="Screenshot 2020-04-29 at 15 52 02" src="https://user-images.githubusercontent.com/47178106/80610591-70e04d00-8a31-11ea-8eed-8de4e994a723.png">

**Comment by briri** [comment]
hmm. We had to restrict the size of both the guidance section and the answer text areas due to the available real estate on the page. 

Would it work to add a horizontal scroll bar to the Tinymce text area? That would allow the user to scroll to see the full table.

I also notice that the Example answer is not staying inside the blue boundaries of its container

**Comment by magdalenadrafiova** [comment]
I personally think that to scroll horizontally on the page is not ideal but could be possibly a temporary fix for now before we do more changes to DMPonline pages in general and just make the windows more wide across the screen if that could be possible in the future? 
any thoughts on this @sjDCC and @mariapraetzellis

**Comment by sjDCC** [comment]
Yes, I think we need a horizontal scroll for now. Or we supersede this with improved layout. I'm inclined to add all guidance and examples below the text box to give more space for that

**Comment by briri** [comment]
ok. Is this something we want to prioritize over the rails5 work? If so we can send the patch directly to the master branch.

I think it will be more efficient to change up the layout of this page when we do the work to change up the contributors, sharing, etc.

**Comment by sjDCC** [comment]
Yes, I think it can wait til after Rails 5.

Good point about limiting width

**Comment by raycarrick-ed** [comment]
Tried this with latest development branch. Added a ridiculously wide table and a horizontal scroll bar appeared for the tinymce box. So maybe this isn't an issue on the latest codebase.

**Comment by dsisu** [comment]
This issue was raised several times during the DMPonline Usability Exercise in 2020.

**Comment by mariapraetzellis** [comment]
We should look into the TineMCE scroll bar- @martaribeiro will look into this.

**Comment by pherterich** [comment]
It does, closing out
